gateway/src/apicast/executor.lua

```diff
@@ -59,6 +63,28 @@ function _M:context(phase)
     return shared_build_context(self)
 end
 
+local init = _M.init
+function _M:init()
+    init(self)
+
+    for _, policy_mod in ipairs(policy_modules) do
+       if policy_mod.init then
+           policy_mod.init()
```

Comment:
I think we'll need to check which policies are in the global policy chain. Otherwise, `init()` might be run twice for those policies. Same for `init_worker()`